src/components/Image.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC } from 'react'
+import styled from '@emotion/styled'
+import Heart from './Heart'
+
+interface ImageProps {
+  url: string,
+  width: number,
+  height: number,
+  favorited: boolean,
+  fromSearch: boolean,
+  onClick(url: string): any,
+}
+
+const Image: React.SFC<ImageProps> = (props) => {
+  const icon = props.favorited ? "redHeartIcon" : "whiteHeartIcon"
+  const alt = props.favorited ? "red heart icon" : "white heart icon"
+  return (
+    <Container>
+      {props.fromSearch ? <Img
+        src={props.url}
+        width={props.width}
+        height={props.height}
+      /> : <ImgFavorite
+      src={props.url}
+      width={props.width}
+      height={props.height}
+      />}
+      {props.fromSearch ? <HeartWrapper onClick={() => props.onClick(props.url)}>
+        <Heart icon={icon} alt={alt} /> </HeartWrapper> : 
+        <HeartWrapperFavorite onClick={() => props.onClick(props.url)}>
+        <Heart icon={icon} alt={alt} />
+      </HeartWrapperFavorite>}
```

Comment:
There is a lot going on here with the ternaries, can you think of a more readable way to write this `jsx`?